bazel/repositories.bzl

```diff
@@ -91,19 +91,27 @@ cc_library(
     visibility = ["//visibility:public"],
 )
 
+# There is some moderate evil here. buildconf needs to be invoked in its source
+# location, where it also generates temp files (no option to place in TMPDIR).
+# Also, configure does a cd relative dance that doesn't play nice with the
+# symlink execroot in the Bazel build. So, copy everything prior to building.
+ARES_CONFIG_SH_CMDS = [
+    "export TMPDIR=$$(realpath $(@D))",
```

Comment:
What is the reason to point TMPDIR to a path under `$(@D)` instead of real TMPDIR?